fe/src/components/Header/Header.tsx

```diff
@@ -23,65 +23,42 @@ const NAV_ITEMS = [
 ];
 
 export default function Header(): JSX.Element {
-  const { isOpen, onOpen, onClose } = useDisclosure();
-
   return (
-    <Box bg={useColorModeValue('gray.100', 'gray.900')} px={4}>
+    <Box
+      bg={useColorModeValue('gray.100', 'gray.900')}
+      px={4}
+      position="sticky"
+      zIndex={3}
+      top={0}
+    >
       <Flex h={16} alignItems="center" justifyContent="space-between">
-        <IconButton
-          size="md"
-          icon={isOpen ? <CloseIcon /> : <HamburgerIcon />}
-          aria-label="Open Menu"
-          display={{ md: !isOpen ? 'none' : 'inherit' }}
-          onClick={isOpen ? onClose : onOpen}
-        />
         <HStack spacing={8} alignItems="center">
-          <Heading as="h4" size="md">TPT-Loane</Heading>
-          <HStack
-            as="nav"
-            spacing={4}
-            display={{ base: 'none', md: 'flex' }}
-          >
+          <Drawerbar />
+          <Heading as="h4" size="md">
+            TPT-Loane
+          </Heading>
+          <HStack as="nav" spacing={4} display={{ base: 'none', md: 'flex' }}>
```

Comment:
Something is different about breakpoints when changing from normal sidebar to drawer sidebar view. Make sure that breakpoints are identical when changes should be made. 

Currently at some point both route links are visible:

![something](https://user-images.githubusercontent.com/48998862/116250444-6789dd80-a776-11eb-8e93-feb393216195.JPG)